src/components/details/DateRangeSlider.tsx

```diff
@@ -14,6 +15,33 @@ import { DatePicker } from "@mui/x-date-pickers/DatePicker";
 import { AdapterDayjs } from "@mui/x-date-pickers/AdapterDayjs";
 import { dateDefault } from "../common/constants";
 
+const datePikerSlotProps = {
```

Comment:
duplicated code from here with `DateRangeFilter`. Please double-check the two files.